velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.cpp

```diff
@@ -31,44 +30,53 @@ using FileSystemMap = folly::Synchronized<
 
 /// Multiple S3 filesystems are supported.
 /// Key is the endpoint value specified in the config using hive.s3.endpoint.
-/// If the endpoint is empty, it will default to AWS S3.
+/// If the endpoint is empty, it will default to AWS S3 Library.
```

Comment:
We should add this to the documentation here: https://github.com/facebookincubator/velox/blob/main/velox/docs/configs.rst#amazon-s3-configuration

Reply:
Since this PR is already imported, I opened a PR here to add this and fix other doc issues I found
https://github.com/facebookincubator/velox/pull/11418/files